spring-core/src/main/java/org/springframework/util/SocketUtils.java

```diff
@@ -38,7 +38,7 @@
  * @author Gary Russell
  * @since 4.0
  */
-public class SocketUtils {
+public abstract class SocketUtils {
```

Comment:
`SocketUtils` is intentionally *not* abstract. 

See the JavaDoc for its constructor. 

Please undo this change accordingly.

Reply:
Thank you for your review. I modified it.

Reply:
You're welcome, and thanks for making the change.

Reply:
I also just noticed that the `ReflectUtils` class you modified is from CGLIB. So I requested additional changes in a separate review.

Sorry for not catching that the first time: I noticed the issue with `SocketUtils` immediately on my phone (b/c I'm an author of that class), but I neglected to see the `cglib` package at that time.